grapher/stackedCharts/StackedBarChart.tsx

```diff
@@ -262,7 +262,7 @@ export class StackedBarChart
                                 </h3>
                             </td>
                         </tr>
-                        {series.map((series) => {
+                        {[...series].reverse().map((series) => {
```

Comment:
Both `[...series].reverse()` and `series.slice().reverse()` work correctly, but I'm not sure if one of them is better.

Reply:
👌 Either is fine